packages/fxa-auth-server/config/index.js

```diff
@@ -583,6 +583,28 @@ const conf = convict({
     env: 'DEVICE_NOTIFICATIONS_ENABLED',
     default: true
   },
+  // A safety switch for disabling new signins/signins from particular clients,
+  // as a hedge against unexpected client behaviour.
+  oauthClientsSigninDisabled: {
```

Comment:
> Is there a meaningful difference between this and disabledClients on the OAuth server? 

The intention is for this one to prevent "new connections" from the specified clients while continuing to operate normally for users who have already connected them; the oauth-server one refuses service to all instances of the specified clients, even if they have already connected to the user's account.

I don't think we ever settled on a really good name for the combination of signup/signin, but I'm happy to take suggestions. I think it needs to be named differently *somehow* to distinguish its slightly later touch from the similar flag on the oauth-server.

Reply:
What do you think about `oauth.disableNewClientConnections`?